docs/concepts/about_biome.md

```diff
@@ -0,0 +1,17 @@
+# About Biome
+
+Biome is a module in libsplinter containing several sub modules that provide
+support for user management, user credential management, and private key
+management. These submodules and their functionality are bundled into Biome’s
+rest api module and can be integrated into an existing splinter rest api using
+libsplinter’s rest-api framework.
+
+## Modules
+
+* **User Management**: API for adding, deleting, updating, and retrieving user
+information.
+
+* **Credential Management**: API to register and authenticate a user using a
```

Comment:
inconsistent tense here:  The previous built is "API for Xing..." where this and the next item are "API to X" (where X is a verb).